web/src/app/alerts/CreateAlertDialog/StepContent/CreateAlertForm.tsx

```diff
@@ -10,9 +10,10 @@ interface CreateAlertFormProps {
   activeStep: number
   value: Value
 
-  errors: Error[]
+  errors?: Error[]
+  error?: Error
```

Comment:
What's the difference between `errors: [err]` and `error: err` can we stick with just a single prop, or name it differently if it's special?

Reply:
This was a prop that was missed, no changes were made in functionality

Reply:
Going to revert this file